### PR TITLE
Change onTeamsMembersAdded for Teams to use getMember()

### PR DESCRIPTION
**Pull request by EricDahlvang**
Fixes https://github.com/microsoft/botbuilder-js/issues/2068

### CODE REVIEW

File: libraries/botbuilder/src/teamsActivityHandler.ts [hunk 1]
```diff
@@ -401,21 +399,27 @@ export class TeamsActivityHandler extends ActivityHandler {
...
-
-                // if we have the TeamsChannelAccount in our lookup table then overwrite the ChannelAccount with it
-                const teamsChannelAccount = teamsChannelAccountLookup[channelAccount.id];
-                if (teamsChannelAccount !== undefined) {
-                    context.activity.membersAdded[i] = teamsChannelAccount;
+                try {
+                    const teamsChannelAccount = await TeamsInfo.getMember(context, channelAccount.id);
+                    context.activity.membersAdded[i] = teamsChannelAccount;    
+                } catch (err) {
+                    if (err.body && err.body.error && err.body.error.code === 'ConversationNotFound') {
```
[the remark this answers is not in the document]
Comment (stevengum) [reply libraries/botbuilder/src/teamsActivityHandler.ts:410]: Once we move to a newer version of TypeScript this type of code will become easier.

**Comment by stevengum** [comment]
@EricDahlvang, I added some small feedback, I think the only one worth taking is the one regarding inspection of the err ([link](https://github.com/microsoft/botbuilder-js/pull/2103#discussion_r414925346)). Otherwise it looks good 👍

File: libraries/botbuilder/src/teamsActivityHandler.ts [hunk 2]
```diff
@@ -401,21 +399,27 @@ export class TeamsActivityHandler extends ActivityHandler {
...
+                        // unable to find the member added in ConversationUpdate Activity in the response from the getMember call
+                        const teamsChannelAccount: TeamsChannelAccount = { 
+                            id: channelAccount.id,
+                            name: channelAccount.name,
+                            aadObjectId: channelAccount.aadObjectId,
+                            role: channelAccount.role,
+                        };
```
[the remark this answers is not in the document]
Comment (EricDahlvang) [reply libraries/botbuilder/src/teamsActivityHandler.ts:416]: ha!  ... that's cool!

File: libraries/botbuilder/src/teamsActivityHandler.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (EricDahlvang) [reply libraries/botbuilder/src/teamsActivityHandler.ts]: changed.  thanks for the feedback @stevengum

**Pull request merged**